index.js

```diff
@@ -1,37 +1,77 @@
-const { ApolloServer } = require('apollo-server');
+require('dotenv').config()
```

Comment:
dotenv to be used only at the entry file. Care to remove other ones

Reply:
done